docs/maps/opening-a-website.md

```diff
@@ -59,6 +59,10 @@ If you set `openWebsiteTrigger: onicon`, when the user walks on the layer, an ic
     <figcaption class="figure-caption">The iFrame will only open if the user clicks on icon</figcaption>
 </figure>
 
+### Showing a hint on minimized website
+
+If you close a website it will be minimized and is still accessible on it's icon on the bottom on the screen. Since there is no info except for it's favicon you can set `openWebsiteHint: your hint to the site`. On minimized websites the hint message will be shown on hover.
```

Comment:
```suggestion
If you close a website it will be minimized and is still accessible on its icon on the bottom on the screen. Since there is no info except for its favicon you can set `openWebsiteHint: your hint to the site`. When moving your mouse over the icon, the hint message will be shown.
```